tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -54,22 +70,34 @@ def test_user_path_cli_priority(convert2rhel):
     config = [
         Config(
             "~/.convert2rhel.ini",
-            "[subscription_manager]\nusername = config_username\npassword = config_password\nactivation_key = config_key\norg = config_org",
+            """
+            [subscription_manager]
+            username = config_username
+            password = config_password
+            activation_key = config_key
+            org = config_org
+            """,
         )
     ]
     create_files(config)
 
-    with convert2rhel("--password password --debug") as c2r:
+    with convert2rhel("--username username --password password --debug") as c2r:
         # Found options in config file
-        c2r.expect("DEBUG - Found username in /root/.convert2rhel.ini")
-        c2r.expect("DEBUG - Found password in /root/.convert2rhel.ini")
-        c2r.expect("DEBUG - Found activation_key in /root/.convert2rhel.ini")
-        c2r.expect("DEBUG - Found org in /root/.convert2rhel.ini")
-        c2r.expect(
-            "WARNING - You have passed either the RHSM password or activation key through both the command line and"
-            " the configuration file. We're going to use the command line values."
+        c2r_expect_index = c2r.expect(
+            [
+                "DEBUG - Found username in /root/.convert2rhel.ini",
+                "DEBUG - Found password in /root/.convert2rhel.ini",
+                "DEBUG - Found activation_key in /root/.convert2rhel.ini",
+                "DEBUG - Found org in /root/.convert2rhel.ini",
+                "WARNING - You have passed either the RHSM password or activation key through both the command line and"
+                " the configuration file. We're going to use the command line values.",
```

Comment:
Sadly this will not work as intended. It should return the index of the first string that it finds (https://pexpect.readthedocs.io/en/stable/api/pexpect.html?highlight=expect#pexpect.spawn.expect). With this we are not able to verify that the second line is found. So we need to call expect() multiple times to verify multiple strings appearing in the output

Reply:
I agree with Martin

Reply:
Oh :( 

Okay, I will revert the change